README.md

```diff
@@ -4,3 +4,21 @@ Vulkan users. It is meant to facilitate triaging, discussion and routing of
 issues that don't necessarily fit under the scope of a single project within
 the Vulkan ecosystem.
 
+# Map of Vulkan Ecosystem projects
+Vulkan ecosystem is a big place, but there's a lot out there thats useful! In no particular order...
+
+## RenderDoc
+RenderDoc - a graphics debugger, currently available for Vulkan, D3D11, D3D12, and OpenGL development on Windows 7 - 10 and Linux.
+Contacts: [baldurk@baldurk.org](mailto:baldurk@baldurk.org), [#renderdoc on freenode IRC](https://kiwiirc.com/client/irc.freenode.net/#renderdoc)
+Main site: https://renderdoc.org/
+Code: https://github.com/baldurk/renderdoc
+License:  (MIT)[https://github.com/baldurk/renderdoc/blob/v0.x/LICENSE.md]
+Dependencies: MSVC 2015 / gcc 5 / clang 3.4
+Code of conduct: (contributor covenant)[https://github.com/baldurk/renderdoc/blob/v0.x/CODE_OF_CONDUCT.md]
```

Comment:
Dependencies may change. Also arguably a licence.
I wonder if this is not a maintenance hazard.

Should Dependencies also have a link?

The `v1.x` is supposed to be the master branch. Should links lead there instead?

Reply:
This merits a bit of explanation: we're trying to gather compiler requirements across the various projects, as an initial step to try to ultimately address #8. This is intended to collate that info as we go through the various projects.

I expect this info to eventually move elsewhere, but for now I'd like to track this together with the project list to make things easier to get going.

The intent here is not to have an exhaustive list of software dependencies. I agree that this would be a maintenance nightmare.